SwiftDiffusion/Services/FilePickerService.swift

```diff
@@ -41,7 +41,7 @@ struct FilePickerService {
         if response == .OK, let url = panel.urls.first, url.pathExtension == Constants.FileTypes.shellExtension {
           continuation.resume(returning: url.path)
         } else {
-          print("Error: Selected file is not a .\(Constants.FileTypes.shellExtension) script.")
+          Debug.log("Error: Selected file is not a .\(Constants.FileTypes.shellExtension) script.")
           continuation.resume(returning: nil)
         }
```

Comment:
Suggestion: Provide selected file path in log.

Example case: `/usr/path/to/invalid_name/webui.~sh`